client/connect.go

```diff
@@ -26,13 +26,18 @@ func parseConnectResponse(response *http.Response) (*models.Connect, error) {
 	return nil, makeGenericError(response.StatusCode, bufferStr)
 }
 
-func (c *Client) GetWidget(userGuid string) (*models.Connect, error) {
+func (c *Client) GetWidget(userGuid string, params models.ConnectParams) (*models.Connect, error) {
```

Comment:
To avoid a breaking change here, let's make two methods for this:

```go
func (c *Client) GetWidget(userGuid string) (*models.Connect, error) {
  return c.GetWidgetWithConnectParams(userGuid, nil)
}
```

and

```go
func (c *Client) GetWidgetWithConnectParams(userGuid string, params models.ConnectParams) (*models.Connect, error) {
```

And the old method can simply be a wrapper for the new one, but that let's others use this without needing to add a second param.

And with this we should check perform a `nil` check on params.